packages/office-ui-fabric-react/src/utilities/decorators/withResponsiveMode.tsx

```diff
@@ -92,9 +95,7 @@ export function withResponsiveMode<TProps extends { responsiveMode?: ResponsiveM
     }
 
     private _onResize = () => {
-      const element = findDOMNode(this) as Element;
-      const currentWindow = (element && getWindow(element)) || window;
-      const responsiveMode = getResponsiveMode(currentWindow);
+      const responsiveMode = getResponsiveMode(this.context.window);
```

Comment:
@joschect check with @ThomasMichon that this `window` is correct for embedded scenarios

Reply:
I'm not sure it would be correct by default--I think the new WindowProvider approach relies on the user to correctly set the window for alternative contexts. @dzearing correct me if I'm wrong but that's what I remember from discussions and what's mentioned at the end of [the package docs](https://www.npmjs.com/package/@fluentui/react-window-provider) (though we could stand to make that more explicit).

Reply:
If that's the case, we would need to move the responsive utilities to react-next (and update references) so that we can include specific guidance for consumers to add WindowProviders as part of the release notes.

Reply:
It already wasn't going to be quite right. Because the onResize only triggered based off the global `window`, it would only update the resize when that window changed, rather than when the embedded window changed :/